add section for making your own VEDA instance (#213)
NASA-IMPACT/veda-docs: open-source-ecosystem/creating-your-own-instance/creating-your-own-veda-dashboard.qmd

---
title: How To Make Your Own Dashboard
subtitle: Even if you already have a website, you can use the tools
---

# Using the VEDA Dashboard in Your Own Projects

> VEDA was designed as an **open, accessible, and powerful platform** to help science teams communicate insights hidden in large-scale geospatial data.

Reviewer: ```suggestion
VEDA provides the software architecture for an **open, accessible, and powerful platform** to help science teams communicate insights hidden in large-scale geospatial data.
```

> - Built to work with **cloud-optimized formats** like COGs.

Reviewer: ```suggestion
- Built to work with **cloud-optimized formats** like [COG](https://cogeo.org/) and the emerging web-optimized [Zarr](https://zarr.dev/).
```

- Enables rapid, compelling storytelling at scale.
- Open-source under permissive licenses to encourage broad use and innovation.

Our goal is to lower the barriers to entry for scientists to talk about their work, for users to learn about and explore geospatial data, and for geospatial data to come alive in ways it never has before. 

Whether you want a turnkey dashboard, individual components, or a foundation for your own innovation, we want the VEDA Dashboard to be flexible and modular enough for your needs.

> As VEDA grows, we want to make it straightforward for organizations outside of NASA to use and extend these tools. If you have any feedback on the tools or your ability to use them, please reach out to veda@uah.edu

Reviewer: ```suggestion
As the number of projects that depend on VEDA grows, we want to make it straightforward for organizations outside of NASA to use and extend these tools. If you have any feedback on the tools or your ability to use them, please reach out to veda@uah.edu
```

- This approach gives you full control and flexibility
- Benefit from updates to core VEDA functionality

**When to choose this:**

- You value simplicity and speed.
- You are also using other VEDA tools and patterns, such as STAC and eoAPI.
- You don't already have a web presence for your project, or you are interested in completely replacing your current website

### You already have a website, and want to include parts of the VEDA Dashboard

####3. Use VEDA as an NPM Package (for Component Integration)

If you already have a robust web presence but want to use VEDA's powerful **tools** and **widgets**—such as data stories or analysis tools, then you can simply import them as an NPM package.

- Allows you to selectively integrate VEDA components into your existing site.
- Take advantage of novel storytelling and COG (Cloud-Optimized GeoTIFF) capabilities without a full rebuild.

**When to choose this:**

- You want to enhance your existing website
- You already have a preferred CMS tool for managing your site

---

## Content Management Options

> Content in VEDA is designed to be **easily managed**.

Reviewer: ```suggestion
The VEDA Dashboard is designed for content to be **easily managed**.
```

This document shares guidance on how you can adopt the VEDA Dashboard in a way that fits your needs.

## Getting Started

If you are going to use the VEDA Dashboard for your own purposes, we assume that you will be self-hosting your own website and data. 
- That website and data could all be VEDA tools, so if you're starting from scratch, don't worry! Visit the [top-level instructions](open-source-ecosystem/creating-your-own-instance/index.qmd) for using VEDA.

The primary repository for the VEDA Dashboard is here: 
- [veda-ui Github](https://github.com/NASA-IMPACT/veda-ui)

We have created a Template Dashboard to showcase how you can pull in veda-ui and create a website compiled with NextJS. 
You can explore and review that implementation here:
- [Template Dashboard GitHub](https://github.com/NASA-IMPACT/next-veda-ui)
That Template Dashboard is live here:
- [Template Dashboard](https://veda-ui-next-test.netlify.app/)

## Choosing Your Path: How to Use VEDA

Depending on your goals and technical preferences, there are three main ways to leverage the VEDA Dashboard:

### Do you want the full VEDA Dashboard?

> #### Fork (for Light Customizations)

Reviewer 2: Fork and cloning will do the same job. Fork will retain the repository's git history, but I don't think it is a significant difference that users will care about. 
It would make sense to encourage starting with a copy of the repo (I will say clone it) if light customization. If you want to integrate parts of VEDA UI into the existing web presence, use the modular version of UI.

Author: Okay, that makes sense! I combined the two sections into one, focusing on cloning.
